Meeting Notes/MeetingNotesFromHeinz.md

```diff
@@ -0,0 +1,28 @@
+# Meeting Notes - Code Voyagers
+
+Dates: 06. - 08. November 2023
+Participants: Jacub, Smarí, Tiburcio, Alberto, Heinz
+Location: El Rincon, Las Palmas
+
+## Overview of the three weeks
+We have made a full schedule of the three weeks
+- Jan/Feb: Startup with two or three online icebreaking exercises
+- 26th – 30th Feb: Online week with design sprint – each group put a link to the DS for evaluation Friday at 12:00
+- 4th – 18th March: Two weeks at El Rincon in Las Palmas (Danish expedition dates)
+- Monday 4th March at 18:00: First meetup at El Rincon
+- Tuesday 5th March: Icebreaking team trip. Each team presents a photo with a team name and roles.
+- Saturday 9th March: Common Activity
+- Friday 15th March: Presentation
+
+## Project Scope:
+- A new and more open project description with focus on collaboration, workflow, and a sustainable approach
+- Recommended technologies: React, typescript.
+(teams can individually decide to use Next, NodeJs, MySql, Supabase etc)
+- A strict and detailed github guide that shows each team how to work together with git (Did we finish this?)
```

Comment:
No we did not finish it but we will... hopefully next week there will be a pull request with this guide (maybe in PDF fromat so students can use it as a reference)